types/underscore/index.d.ts

```diff
@@ -3710,9 +3711,13 @@ declare module _ {
         * @keys The key/value pairs to keep on `object`.
         * @return Copy of `object` with only the `keys` properties.
         **/
+        pick<T, K extends keyof T>(obj: T, ...keys: K[]): Pick<T, K>;
         pick(
             object: any,
             ...keys: any[]): any;
+        pick(
+            object: any,
+            keys: any[]): any;
```

Comment:
This new overload certainly shouldn't be added; apart from the fact that it is overly permissive, it doesn't add any information because all call signatures that match this will also match the previous one.

Reply:
Yes it was a typo. Removed it